### DISCUSSION TITLE
filter files being checked with cppcheck?

### DISCUSSION THREAD

**Discussion by jalanh11640** [jbenden/vscode-c-cpp-flylint#172] [Q&A] [answered]
Is there a way to filter files being checked with cppcheck?  In the project I work on, some of the files are auto-generated by a 3rd party tool that we use.  I'd like to not check files that match "*.auto.cpp".

Thanks

**Comment by jbenden** [comment, marked as answer]
Hi,

The setting "c-cpp-flylint.excludeFromWorkspacePaths" might help:

```
{
  "c-cpp-flylint.excludeFromWorkspacePaths": [
    "**/*.auto.cpp"
  ]
}
```

Best regards,
-Joe

**Comment by jalanh11640** [comment]
Thanks!  That works great after I restarted vscode. Sent from my iPhoneOn Sep 30, 2022, at 10:20 AM, Joseph Benden ***@***.***> wrote:﻿
Hi,
The setting "c-cpp-flylint.excludeFromWorkspacePaths" might help:
{
  "c-cpp-flylint.excludeFromWorkspacePaths": [
    "**/*.auto.cpp"
  ]
}

Best regards,
-Joe
